src/main/resources/view/LessonListCard.fxml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.ColumnConstraints?>
+<?import javafx.scene.layout.FlowPane?>
+<?import javafx.scene.layout.GridPane?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.Region?>
+<?import javafx.scene.layout.VBox?>
+
+<HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
+  <GridPane HBox.hgrow="ALWAYS">
+    <columnConstraints>
+      <ColumnConstraints hgrow="SOMETIMES" minWidth="10" prefWidth="150" />
+    </columnConstraints>
+    <VBox alignment="CENTER_LEFT" minHeight="105" GridPane.columnIndex="0">
+      <padding>
+        <Insets top="5" right="5" bottom="5" left="15" />
+      </padding>
+      <HBox spacing="5" alignment="CENTER_LEFT">
+        <Label fx:id="id" styleClass="cell_big_label">
+          <minWidth>
+            <!-- Ensures that the label text is never truncated -->
+            <Region fx:constant="USE_PREF_SIZE" />
+          </minWidth>
+        </Label>
+        <Label fx:id="studentName" text="\$studentName" styleClass="cell_big_label" />
+      </HBox>
+      <FlowPane fx:id="tags" />
+      <Label fx:id="lessonName" styleClass="cell_small_label" text="\$studentPhone" />
+      <Label fx:id="timing" styleClass="cell_small_label" text="\$parentName" />
+      <Label fx:id="price" styleClass="cell_small_label" text="\$parentPhone" />
+      <Label fx:id="capacity" styleClass="cell_small_label" text="\$progress" />
+      <Label fx:id="students" styleClass="cell_small_label" text="\$paymentStatus" />
```

Comment:
I'm not sure if this will affect the functionality of a Lesson in full view, but should the values of the text fields match those of the id fields?